app/src/main/res/home/layout/home_movie_item_layout.xml

```diff
@@ -0,0 +1,96 @@
+        android:id="@+id/movie_item_main_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingBottom="@dimen/spacing_medium"
+        android:background="@color/colorPrimaryDark">
+
+        <ImageView
+            android:id="@+id/movie_item_image_backdrop"
+            android:layout_width="match_parent"
+            android:layout_height="150dp"
+            android:scaleType="centerCrop"
+            tools:src="@drawable/ms_top" />
+
+        <ImageView
+            android:id="@+id/movie_item_image_photo"
+            android:layout_width="90dp"
+            android:layout_height="120dp"
+            android:layout_alignParentBottom="true"
+            android:layout_marginBottom="@dimen/spacing_medium"
+            android:layout_marginLeft="@dimen/spacing_medium"
+            android:scaleType="fitXY"
+            tools:src="@drawable/ms_poster" />
+
+        <TextView
+            android:id="@+id/movie_item_text_name"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/movie_item_image_backdrop"
+            android:layout_marginLeft="@dimen/spacing_medium"
+            android:layout_marginRight="@dimen/spacing_small"
+            android:layout_toLeftOf="@+id/movie_item_layout_rate"
+            android:layout_toRightOf="@+id/movie_item_image_photo"
+            android:maxLines="1"
+            android:textColor="@color/white"
+            android:textSize="16sp"
+            tools:text="Missão Impossível" />
+
+        <ImageView
+            android:id="@+id/movie_item_image_calendar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignLeft="@+id/movie_item_text_name"
+            android:layout_below="@+id/movie_item_text_name"
+            android:src="@drawable/ic_calendar" />
+
+        <TextView
+            android:id="@+id/movie_item_text_year"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/movie_item_text_name"
+            android:layout_marginLeft="@dimen/spacing_small"
+            android:layout_toRightOf="@+id/movie_item_image_calendar"
+            android:textColor="@color/white"
+            android:textSize="16sp"
+            tools:text="2017" />
+
+        <LinearLayout
+            android:id="@+id/movie_item_layout_rate"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentRight="true"
+            android:layout_below="@+id/movie_item_image_backdrop"
+            android:layout_alignBottom="@+id/movie_item_text_year"
+            android:layout_marginRight="@dimen/spacing_medium"
+            android:gravity="center">
+
+            <TextView
+                android:id="@+id/movie_item_text_rate"
```

Comment:
It's considered good practice to use a TextView's `drawable` for situations like this. If you used its endDrawable for the favorite icon, you'd avoid both the ImageView and the LinearLayout.

Reply:
Yes! you're right!